android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -503,7 +503,7 @@ private boolean removeOrAnimateExitRecursive(View view, boolean shouldRemove) {
         View child = viewGroup.getChildAt(i);
         if (removeOrAnimateExitRecursive(child, shouldRemove)) {
           hasAnimatedChildren = true;
-        } else if (shouldRemove) {
+        } else if (shouldRemove && child.getId() != -1) {
```

Comment:
Sorry for this basic question but could you please explain in which cases `getId()` returns -1?

Reply:
This means that the view is native only, and not a react one.